docs/configuration.asciidoc

```diff
@@ -24,7 +24,7 @@ When using this method, strings are split by comma, e.g.,
 [[configuration-precedence]]
 === Configuration precedence
 
-Options are applied in the following order (last one wins):
+Options are applied in the following order (last one wins): 
```

Comment:
Should this say listed in descending order with the last taking precedence for consistency with other articles?